src/py/flwr/cli/run/run.py

```diff
@@ -102,10 +102,17 @@ def on_channel_state_change(channel_connectivity: str) -> None:
         """Log channel connectivity."""
         log(DEBUG, channel_connectivity)
 
+    if cert_path := federation.get("root-certificates"):
+        root_certificates = Path(cert_path).read_bytes()
```

Comment:
In other parts of the codebase, we use `root_certificates` to refer to the path, and `root_certificates_bytes` to refer to the content

Reply:
Sounds good, I'll update it